components/filter_comments/README.md

```diff
@@ -0,0 +1,47 @@
+# Filter comments
+
+### Description
+Component that filters code based on the code to comment ratio
+
+### Inputs/Outputs
+
+**The component comsumes:**
+- code
+  - content: string
+
+**The component produces:**
+
```

Comment:
Probably annoying to implement but can we mention that the component does not produce any subsets instead of keeping it blank?